jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -145,10 +146,17 @@ def _on_dataset_selected_changed(self, event={}):
         if self.config != "cubeviz":
             return
         # self.dataset might not exist when app is setting itself up.
-        if hasattr(self, "dataset") and self.dataset.selected_dc_item.ndim > 2:
-            self.is_cube = True
-        else:
-            self.is_cube = False
+        if hasattr(self, "dataset"):
+            if isinstance(self.dataset.selected_dc_item, list):
+                datasets = self.dataset.selected_dc_item
+            else:
+                datasets = [self.dataset.selected_dc_item]
+            for dataset in datasets:
+                if dataset.ndim > 2:
+                    self.is_cube = True
+                    break

```

Comment:
this assumes either _all_ cubes or no cubes, which should be safe for now because we only allow aperture photometry on the flux cube cubeviz (and not collapsed cubes or imported images).  Maybe it would just be worth adding a comment here in case that assumption ever changes?

Reply:
Added.